settings/Core.setting.php

```diff
@@ -192,6 +192,21 @@
     'description' => 'If enabled, CiviCRM will permit submissions from external sites to profiles. This is disabled by default to limit abuse.',
     'help_text' => NULL,
   ),
+  'dismiss_info_alerts' => array(
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'dismiss_info_alerts',
+    'type' => 'Boolean',
+    'quick_form_type' => 'YesNo',
+    'default' => TRUE,
+    'html_type' => 'radio',
+    'add' => '4.7',
+    'title' => 'Dismiss info alerts',
```

Comment:
I think success alerts expire at the moment too. What about saying 'Allow alerts to auto-dismiss?'.

Since we are affecting more than just info alerts, the name of the setting should change too. How about changing from dismiss_info_alerts to allow_alert_autodismissal ?